PatientRegistrationApp/src/components/HouseholdContactInfo.vue

```diff
@@ -119,8 +103,7 @@ import EventBus from '../eventBus'
     name: 'SinglePatientContactInfo',
 	data () {
 		return { 
-			phonetype:['Home', 'Mobile', 'Work'],
-			radios: 'May we contact you for a follow up vaccination?',
+			phoneTypeOptions:['Home', 'Mobile', 'Work'],
```

Comment:
Have we ever thought why we have the phone number type input but don't require it?  question if we really even need that input...

Reply:
It would be nice to know if texting is an option (if "mobile").  I don't know if we should make it required, but it could be a nice-to-have bit of information for follow-up notifications.